lib/src/progressBar/progressBar.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="progress" [ngClass]="className" [attr.id]="id" [ngStyle]="height ? { height: height + 'px' } : null">
```

Comment:
the only style you want to use here is the height
```suggestion
<div class="progress" [ngClass]="className" [attr.id]="id" [style.height.px]="height ? height : null">
```

Reply:
Make sense.